example/taglist.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
Maybe this file shouldn't be pushed this has nothing to do with tscat_gui code, or at least it should use tscat_gui flow layout not its own copy.

Reply:
I'll get rid of it.